inventory.py

```diff
@@ -26,6 +26,8 @@ def parse_params(logger):
                       action="store_true", dest="loopback")
     parser.add_option("-s", "--sleep", help="specifiy sleep time for inventory builder, default: 1800", action="store",
                       dest="sleep")
+    parser.add_option("-t", "--timeout", help="specifiy timeout for fetching data from vROps, default: 600", action="store",
+                      dest="timeout")
```

Comment:
Is there a difference to `--sleep`?

Reply:
it is. it defines how much time we give the inventory to fetch individual target vrops instances. If it is a bigger AZ, it takes longer for the inventory to fetch the data.
This is more like a failsafe, if something goes wrong in the thread, we kill it after $timeout seconds